pyiron/atomistics/structure/analyse.py

```diff
@@ -130,4 +130,91 @@ def get_layers(self, distance_threshold=0.01, id_list=None, wrap_atoms=True, pla
             return np.asarray(layers).flatten()
         return np.vstack(layers).T
 
+    def pyscal_steinhardt_parameter(self, neighbor_method="cutoff", cutoff=0, n_clusters=2,
+                                            q=(4, 6), averaged=False, clustering=True):
+        """
+        Calculate Steinhardts parameters
+
+        Args:
+            job (job): pyiron job
+            neighbor_method (str) : can be ['cutoff', 'voronoi']
+            cutoff (float) : can be 0 for adaptive cutoff or any other value
+            n_clusters (int) : number of clusters for K means clustering
+            q (list) : can be from 2-12, the required q values to be calculated
+            averaged (bool) : If True, calculates the averaged versions of the parameter
+            clustering (bool) : If True, cluster based on the q values
+
+        Returns:
+            q (list) : calculated q parameters
+
+        """
+        from pyiron.atomistics.structure.pyscal import get_steinhardt_parameter_structure
+        return get_steinhardt_parameter_structure(
+            structure=self._structure, neighbor_method=neighbor_method, cutoff=cutoff, n_clusters=n_clusters,
+            q=q, averaged=averaged, clustering=clustering
+        )
+
+    def pyscal_cna_adaptive(self, mode="total", ovito_compatibility=False):
+        """
+        Use common neighbor analysis
+
+        Args:
+            atoms (pyiron.structure.atoms.Atoms): The structure to analyze.
+            mode ("total"/"numeric"/"str"): Controls the style and level
+                of detail of the output.
+                - total : return number of atoms belonging to each structure
+                - numeric : return a per atom list of numbers- 0 for unknown,
+                    1 fcc, 2 hcp, 3 bcc and 4 icosa
+                - str : return a per atom string of sructures
+            ovito_compatibility(bool): use ovito compatiblity mode
+
+        Returns:
+            (depends on `mode`)
+        """
+        from pyiron.atomistics.structure.pyscal import analyse_cna_adaptive
+        return analyse_cna_adaptive(atoms=self._structure, mode=mode, ovito_compatibility=ovito_compatibility)
+
+    def pyscal_centro_symmetry(self, num_neighbors=12):
+        """
+        Analyse centrosymmetry parameter
+
+        Args:
+            atoms: Atoms object
```

Comment:
If use 
```
:class:`.Atoms`
```
or

```
:class:`pyiron.structure.atoms.Atoms`
```

Sphinx turns this into links automatically.

Reply:
Thanks for the explanation! Actually, I did not yet have any corrections of the Docstrings included - there are some wich have to much arguments listed...
Is there actually a way to copy Docstrings? For now, I copied Docstrings as they are which is a dublication of code.

Reply:
I figured that you just copied them, but since you're already touching them I thought I point it out. ;)

@samwaseda used something like this

https://github.com/pyiron/pyiron/blob/182f47289332d1b4bba51beef732e9f3841985d7/pyiron/atomistics/structure/atoms.py#L1169

to avoid copying the strings by hand.

Reply:
Sure, updating them is on the list :) I will use the copy __doc__ in the deprecated methods.